docs/24pullrequests.md

```diff
@@ -18,4 +18,4 @@ Something missing?  Create a pull request!
 
 Check out the [Getting Involved in a Track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md) documentation, and some [ideas for things we are looking for](http://exercism.io/languages/ruby/todo) for exercises that are needed to be implemented.
 
-We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/xruby](https://badges.gitter.im/exercism/xruby.svg)](https://gitter.im/exercism/xruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/xruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```

Comment:
Can you rebase against master?  I think these changes are already in place, and rebasing on master should end up removing this change from this PR.

Reply:
I did and unless I made a mistake, the changes wasn't in place.

Reply:
It was me misidentifying the file as README in project root.  So not used to looking at these in the browser.  When I finally checked it locally, it was glaring me right in the eye.